docs/concepts/withdrawals.md

```diff
@@ -66,8 +66,7 @@ of Capella), there's no mechanism for recycling the validator ID that has been e
 From the first Capella slot, block proposers provide up to 16 withdrawals per
 proposed block.
 
-Proposers start from validator 1, and find validators that qualify: must have an
-Ethereum address as a withdrawal address, must have an excess balance, or have exited.
+Proposers start with validator 1 and search for validators that meet any of these criteria: they have an Ethereum address as a withdrawal address, possess an excess balance, or have exited.
```

Comment:
Maybe... 

Proposers start with validator 1 and locate validators that meet any of these criteria: 
- An Ethereum address as a withdrawal address
- An excess balance
- Have exited

These edits are my impressions w/o actually using the product.